elections/ArviZ_2020.md

```diff
@@ -0,0 +1,58 @@
+# Election 2020
+
+## Summary of Process
+This is the first election cycle for ArviZ. During this cycle we are aiming
+for the following outcomes
+
+* Ratification of GOVERNANCE.md
+* Identification and listing of Core Contributors
+* Nominations for Council Candidates
+* Council Elections. Particularly
+  * Size of Council
+  * Inaugural ArviZ Council
+
+This being the first election means that the process will have more steps than
+the ones listed in GOVERNANCE.md.
+
+The moderators for these tasks live in the PST and CST timezone so the dates are reflective of their localities.
```

Comment:
What does this mean? What timezone is used?

Can we have utc times also visible?

Reply:
It basically means when I say a date it's that date in the American timezones. So I might do something June 8th at 8 pm my time after work, but for you guys it will already be June 9th